public/css/main.css

```diff
@@ -13,25 +13,49 @@ h1 {
 	margin-bottom: 2em;
 }
 
-h3 {
+h2, h3 {
 	font-weight: normal;
 }
 
 p {
 	font-weight: 300;
 }
 
+a {
+    text-decoration: underline;
+    color: inherit;
+}
+
+footer {
+	border-top-style: solid;
+	border-color: #FFF;
+	border-width: 2px;
+	margin-top: 2em;
+	text-align: center;
+	padding: 1em;
+	font-size: 80%;
+}
+
+footer ul {
+	list-style-type: none;
+	padding-left: 0em;
+}
+
+footer ul li {
+	display: inline;
+	margin-right: 1em;
```

Comment:
This will cause your list to be slightly off-center because the last element will have a margin-right as well.
I like to use this type of selector to prevent that:
```
footer ul li+li {
	margin-left: 1em;
}
```